unit-tests/song-services.test.js

```diff
@@ -0,0 +1,64 @@
+const songServiceFunctions = require('../songbank-backend/song-services.js');
+
+var testPlayListTitle = 'Playlist 1'
+
+var testSongTitle = 'Easy On Me'
+
+var testPlaylist = {
+    title: "Test Playlist",
+    songs: [{
+        title: 'Easy On Me',
+        Artist: 'Adele',
+        runtime: 3.45,
+    }]
+  }
+var testPlaylistJson = JSON.stringify(testPlaylist);
+
+
+test('Testing addPlaylist()', () => {
+    const result = songServiceFunctions.addPlaylist(testPlaylistJson);
+    expect(result).toBeTruthy();
+});
+
+
+test('Testing findPlaylistByTitle()', () => {
+    const result = songServiceFunctions.findPlaylistByTitle(testPlayListTitle)
+    expect(result).toBeTruthy();
+});

```

Comment:
Even though we retrieve information, its interesting to see we can just use boolean values to test our code still